.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+Please make sure in case you are changing the spec all our customers are informed beforehand.
+
+You can throw a message in #sig-customer with the changes and the Solution Engineers will take care of
+spread the changes within the customers.
```

Comment:
`spread the changes within the customers` -> `sharing the changes with the customers`